src/sentry/replays/lib/new_query/parsers.py

```diff
@@ -7,6 +7,12 @@
 
 from sentry.replays.lib.new_query.errors import CouldNotParseValue
 
+NULL_VALUES: tuple[str] = (
+    "None",
+    "none",
```

Comment:
I don't think these two should be special cases

Here are some examples of what the user can input, and what it looks like in the url/query param. Once it's in the url it becomes of type `string`. By special casing `"None"` and `"none"` we're treating those `string` values as equivalent to the string representation of pythons `None` value. 

If we special case these then they should be in the docs, because, especially in context of replay, our users are not all python users familiar with this keyword. It's a put dealing with upper & lower case characters and all this stuff. 
So I don't think we should special case these at all. 

![SCR-20241004-lmsw](https://github.com/user-attachments/assets/1a78e5e1-3bea-4e2b-8b7f-0d7ff1a039a0)
![SCR-20241004-lmvb](https://github.com/user-attachments/assets/56fd2541-2f27-48c2-a579-e7218da44adc)
![SCR-20241004-lmxb](https://github.com/user-attachments/assets/f181f509-2f7b-4c57-b02c-71892c414998)